extensions/stats/plugin.cc

```diff
@@ -36,6 +36,13 @@ namespace Plugin {
 
 namespace Stats {
 
+// TODO(PiotrSikora): move this back to Proxy-WASM SDK.
```

Comment:
This got moved already. Care to update it to use Wasm::Common::TrafficDirection?

Reply:
Done.